content/docs/command-reference/data/status.md

```diff
@@ -0,0 +1,123 @@
+# data status
+
+Show changes in data tracked by DVC between the last commit, the dvcfiles and
+the workspace.
+
+## Synopsis
+
+```usage
+usage: dvc data status [-h] [-q | -v]
+                       [--granular] [--unchanged]
+                       [--untracked-files [{no,all}]]
+                       [--json]
+```
+
+## Description
+
+Shows changes that have differences between the last commit (`HEAD`) and
+committed data specified in [metafiles](/doc/user-guide/project-structure)
+files, uncommitted changes that have differences between that specified in the
+metafiles and the <abbr>workspace</abbr>, and the changes that are not in the
+cache but is specified in those metafiles.
+
+The first change shows what is already committed to dvc via `dvc add` or
+`dvc commit` or `dvc repro` but the metafiles are not committed to git yet. The
+second change shows the new changes to the data that is not tracked by DVC yet
+or what you could `dvc commit`. The third change shows the files that are
+missing from the cache. Some of those could be pulled from the remote.
+
+Using `--untracked-files`, it'll show the files that are not being tracked by
```

Comment:
Q: `git status` by default includes Untracked. Should we do the same? Is it a performance optimization?

Reply:
No, I urged to leave untracked files off by default. Please see the discussion in https://github.com/iterative/dvc/pull/7943#issuecomment-1172604913 and let me know if you have concerns.

Reply:
> Is it a performance optimization?

It's no longer be, or will no longer be of performance concern.
We didn't have `--untracked-files=normal` support in dulwich so we had to use `--untracked-files=all`, so it'd be noisy and slow if you have too many untracked files.

But thanks to @dtrifiro's great work in pygit2, the `git.status()` is about 30x faster.
Plus, we now have `--untracked-files=normal` support in pygit2, which will be much faster even for a very large repository. So there's no performance issue now. See https://github.com/iterative/scmrepo/pull/118.

(I have been testing a repo with 150,000 untracked files from a dataset, and `--untracked-files=normal` takes ~20ms now)

Reply:
@skshetry I thought we can't mix pygit2 and dulwich?

Reply:
@dberenbaum, as far as I can tell, it should not be a problem for `dvc data status` as it only does one single `status()` call (also `get_rev()`, but it is a very small utility).

It's only a problem when we are using different APIs that are reading and writing using different backends, eg: `exp run`/`exp show`, etc.

Reply:
Thanks @skshetry! Opened https://github.com/iterative/dvc/issues/8061 to track it and for any further discussion.

Reply:
Resolving this discussion and let's discuss this on https://github.com/iterative/dvc/issues/8061.